enterprise/utils.py

```diff
@@ -1660,14 +1660,15 @@ def customer_admin_enroll_user(enterprise_customer, user, course_mode, course_id
 def get_create_ent_enrollment(
         course_id,
         enterprise_customer_user,
+        enterprise_enrollment_source,
```

Comment:
Wow so no existing test for `get_create_ent_enrollment` fails when we add this? I have not checked but sure would be nice to have a unit test that tested this on its own (if too large scope for your PR we can file it away but if easy may not be a bad opportunity to add at least this one case!)

Yeah on quick check I don't see this being called from _any_ test, which makes me sad

Reply:
I think we have a good opp. to add one if it's missing... please push back if this extends scope in a large way!